frontend/server/src/main/java/org/pytorch/serve/wlm/WorkLoadManager.java

```diff
@@ -85,13 +85,24 @@ public int getNumRunningWorkers(ModelVersionName modelVersionName) {
         return numWorking;
     }
 
+    public boolean isLauncherRestartWorkers(int currentWorkers) {
+        boolean isRestart;
+        if (configManager.isCPULauncherEnabled() && currentWorkers > 0) {
```

Comment:
checks cpu_launcher is enabled and `currentWorkers` is greater than 0 (i.e., only restarts workers when scaling workers up/down; if `currentWorkers==0`, then workers are being initialized)